docs/guides-agent-app.md

```diff
@@ -0,0 +1,506 @@
+
+Later on, you'll be asked to [set three parameters](https://wiki.aragon.org/tutorials/Aragon_User_Guide/#214-set-the-parameters-of-your-democracy-organization-then-click-next) for your organization -- the **support**, the **minimum acceptance quorum**, and the **vote duration**.
+
+We'll go with the following (sensible) defaults:
+
+- Support: 100%
+- Min. Quorum: 0%
+- Duration: 168 hours (or 1 week)
+
+## 1. Install aragonCLI
+
+The [aragonCLI](https://hack.aragon.org/docs/cli-intro) (Command Line Interface) is what we use to create, interact with, and develop Aragon apps.
+
+If you haven't done so already, install it from NPM by running the following command:
+
+`npm install -g @aragon/cli`
+
+Hopefully, it installed successfully 😊.
+
+ If not, we recommend you take a quick look at the [installing aragonCLI](/docs/guides-faq.md#installing-aragoncli) section of our [troubleshooting guide](/docs/guides-faq.md#installing-aragonCLI). It should help diagnose and fix the problem 🧐.
+ 
+If that still doesn't fix things 😟, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help). We're more than happy to help.
+
+Note that aragonCLI v5.4.0 was the first version to [include support](https://blog.aragon.one/aragon-agent-beta-release/) for Agent.
+
+
+If you're unsure which version of aragonCLI you have, run:
+
+`aragon -v`
+
+If your version number is less than 5.9.5, or if it's been a while since you last installed the CLI, we recommend you reinstall it (by running the `npm install` command above).
+
+
+## 2. Install the Agent app
+
+Now that we've installed aragonCLI 🎉, we're ready to install the [Agent app](https://blog.aragon.one/aragon-agent-beta-release/).
+
+aragonCLI installs the [`aragon dao`](https://hack.aragon.org/docs/cli-dao-commands) commands. We use these to interact directly with our DAO from the command line. They're also available directly using the `dao` shortcut.
+
+We'll use the the [`dao install`](https://hack.aragon.org/docs/cli-dao-commands#dao-install) command to install the Agent app.
+
+`dao install` takes two arguments:
+
+1. The address or [aragonID](https://github.com/aragon/aragon-id) name of an Aragon DAO.
+2. The package name of an Aragon app published to [aragonPM](/docs/apm-intro) (for the Agent app this would be agent  or agent.aragonpm.eth).
+`
+
+So in our case, to install the Agent app, we need to run: 
+
+```
+dao install <your organization name> agent --environment aragon:rinkeby --apm.ipfs.rpc https://ipfs.eth.aragon.network/ipfs/
+```
+
+You should see that after `dao install <your organization's name> agent` we pass in two [global options](https://hack.aragon.org/docs/cli-intro.html#global-options): `--enviroment` and `--apm.ipfs.rpc`.
+
+<details>
+<summary>Tell me more about these global options.</summary>
```

Comment:
Here we could also mention that as an alternative is possible to ran `aragon ipfs` on other terminar instead of the `apm.ipfs.rpc` option.